index.js

```diff
@@ -74,6 +53,16 @@ class BabelCompiler {
     this.options.filename = file.path;
     this.options.sourceFileName = file.path;
 
+    if (!this.options.presets) {
+      const babelConfig = new babel.OptionManager().init(this.options);
+      console.log(babelConfig);
+      const hasConfig = babelConfig.presets && babelConfig.presets.length > 0;
+
+      if (!hasConfig) {
+        this.options.presets = ['latest'];
```

Comment:
@denysdovhan If user define `targets` and add something from `latest`, we could replace latest with `env` and set targets. If user define targets + ]es2015], we could use env and exclude plugins from `es16-17` presets. Maybe all this replacements must be accompanied by warnings, I don't know 😄